.github/workflows/build-windows.yml

```diff
@@ -22,8 +22,12 @@ jobs:
       run: |
         curl.exe -o ./apache-maven-3.9.9-bin.zip https://dlcdn.apache.org/maven/maven-3/3.9.9/binaries/apache-maven-3.9.9-bin.zip
         tar -xf ./apache-maven-3.9.9-bin.zip
+    - name: Use develocity extensions
+      run: cp .mvn/develocity-extensions.xml .mvn/extensions.xml
     - name: Build with Maven
       # qa skips documentation - we check it on Jenkins CI
       # We skip checkstyle too - we check it on Jenkins CI
       run: ./apache-maven-3.9.9/bin/mvn -B -e clean install -Pstaging -Pqa '-Dcheckstyle.skip=true'
+      env:
+        DEVELOCITY_ACCESS_KEY: ${{ secrets.DEVELOCITY_API_TOKEN }}
```

Comment:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow

> With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.

So this seems to be almost useless here. (Higher restriction than for Jenkinsfile and CBI).

Reply:
Hmm, that's true. Are you (the project maintainers) also always using forks?

Reply:
Usually PRs from maintainers also come from forks. (With exception of after-release merge for branch in origin).

Reply:
In that case, this configuration can indeed be removed 👍